mona_sdk/authentication.py

```diff
@@ -274,6 +274,10 @@ def _refresh_token(mona_client):
     """
     refresh_token_key = _get_token_info_by_api_key(mona_client.api_key, REFRESH_TOKEN)
     response = _request_refresh_token_with_retries(refresh_token_key, mona_client)
+
+    if not response.ok:
+        response = _request_access_token_with_retries(mona_client)
```

Comment:
please add comment about this case - if the refresh of the token fails we implicitly return the response for the initial request access and it should work fine?

Reply:
Yes it should - the responses from refresh token request and first token request are the same (new token info). I guess that if refresh_token failed then getting a new one can also fail, we don't really know what exactly happened with exceed so i'm not sure why would refresh token fail to begin with... 

I added a log to this case, lmk if it looks ok (do you think a comment is still needed?)

Reply:
its fine now :-)